source/_integrations/plugwise.markdown

```diff
@@ -30,115 +30,103 @@ ha_platforms:
 
-This will also remove all connected Adam devices (such as Anna, Tom or Lisa) or connected Adam/Stretch plugs.
+Each device will list entities such as `binary sensors`, `sensors`, etc., depending on its capabilities: for instance centralized measurements such as `power` for a P1, `outdoor_temperature` on Anna or Adam will be assigned to your gateway device. Heating/cooling device measurements such as `boiler_temperature` will be assigned to the OpenTherm/OnOff device.
 
-### Actions
+## Platform information
 
-#### Update Smile data
+### Climate
 
-Forced update of data from your Smile can be triggered by calling the generic `homeassistant.update_entity` action with your Smile entity as the target.
+The [climate entity](/integrations/climate) is displayed for each zone that includes a thermostat. This can be any supported single thermostat such as the Anna or another type of wired-thermostat, Jip or Lisa combined with one or more Tom/Floor devices. Or a combination of for instance a Lisa, two Jips and several Toms in one large zone.
 
-```yaml
-# Example script change the temperature
-script:
-  force_adam_update:
-    sequence:
-      - action: homeassistant.update_entity
-        target:
-          entity_id: climate.living_room
-```
-
-#### Reboot the Plugwise gateway
-
-action: `button.press`
-
-```yaml
-# Example script change the thermostat schedule
-script:
-  reboot_gateway:
-    sequence:
-      - action: button.press
-        target:
-          entity_id: button.adam_reboot
-```
-
-#### Set HVAC mode
+#### Setting the HVAC mode
 
 action: `climate.set_hvac_mode`
 
 Available options include `off` (Adam only) `auto`, `cool`, `heat`, and `heat_cool` (Anna with Elga only).
 
-The meaning of `off` is that the Adam regulation is set to off. This means that the connected HVAC-system does not heat or cool, only the domestic hot water heating function, when available, is active.
-
-The meaning of `cool` or `heat` is that there is no schedule active. For example, if the system is manually set to cooling- or heating-mode, the system will be active if the room temperature is above/below the thermostat setpoint.
-
-The meaning of `heat/cool` is that there is no schedule active. For example, if the system is in automatic cooling- or heating-mode, the active preset or manually set temperature is used to control the HVAC system.
-
-The meaning of `auto` is that a schedule is active and the thermostat will change presets/setpoints accordingly.
+|HVAC mode|Indication|Description|
+--- | --- | ---
+|`auto` | Active schedule | The thermostat will change presets/setpoints accordingly. |
```

Comment:
Let's avoid the use of tables with longer content, they will be impossible to read on mobile.